Makefile

```diff
@@ -1,2 +1,38 @@
-shell:
+
+# Help message borrowed from https://github.com/edx/devstack, which borrowed it from https://github.com/pydanny/cookiecutter-djangopackage.
+help: ## display a help message
+	@echo $(MAKEFILE_LIST)
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[\.%a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+shell: ## run a shell on the studio-frontend container
 	docker exec -it dahlia.studio-frontend /bin/bash
+
+dev.up: ## bring up studio-frontend container
```

Comment:
Personal opinion: I'd rather save the extra keystrokes and just have `up` instead of `dev.up` for all of these commands

Reply:
That's a good idea. I found it cumbersome too. I'll fix that.